tests/Invoke-DbaQuery.Tests.ps1

```diff
@@ -60,11 +66,11 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
         foreach ($result in $results) {
             $result.TestColumn | Should -Be 'hello'
         }
-        'tempdb' | Should -Bein $results.dbname
+        'tempdb' | Should -BeIn $results.dbname
     }
     It "stops when piped databases and -Database" {
         $dbs = Get-DbaDatabase -SqlInstance $script:instance2, $script:instance3
-        { $dbs | Invoke-DbaQuery -Query "Select 'hello' as TestColumn, DB_NAME() as dbname" -Database tempdb -EnableException } | Should Throw "You can't"
+        { $dbs | Invoke-DbaQuery -Query "Select 'hello' as TestColumn, DB_NAME() as dbname" -Database tempdb -EnableException } | Should -Throw "You can't$PesterV5Throw"
```

Comment:
I'm not fond of the way this looks and would just soon either switch everything to Pester 5 or hold off and leave it all at Pester 4.

Reply:
or we can just `Should -Throw`. In any case leaving this as "pester4-style" isn't a big deal, the move to pester5 will just have to append/prepend a *